docs/words/examples/cursesclient.py

```diff
@@ -182,12 +182,18 @@ def close(self):
         curses.echo()
         curses.endwin()
 
+    def __enter__(self):
+        return self
+
+    def __exit__(self, exc_type, exc_value, traceback):
+        self.close()
+
+
 if __name__ == '__main__':
     stdscr = curses.initscr() # initialize curses
-    screen = Screen(stdscr)   # create Screen object
-    stdscr.refresh()
-    ircFactory = IRCFactory(screen)
-    reactor.addReader(screen) # add screen object as a reader to the reactor
-    reactor.connectTCP("irc.freenode.net",6667,ircFactory) # connect to IRC
-    reactor.run() # have fun!
-    screen.close()
+    with Screen(stdscr) as screen:  # create Screen object
```

Comment:
I think this example should use [`twisted.internet.task.react`](https://twistedmatrix.com/documents/current/api/twisted.internet.task.react.html) instead.  That would make the use of `Screen` as a context manager difficult.  

If anybody else agrees, drop a note here and I'll open a new ticket to address this.